inductiva/utils/visualization.py

```diff
@@ -30,8 +31,9 @@
     "figure.dpi": 100,
 }
 
+
 def create_movie_from_view(view,
-                           output_path='movie.mp4',
+                           output_path="movie.mp4",
```

Comment:
I think this needs to be more explicit, since here we are not aware what a view is here.